db/seeders/report_type_seeder.rb

```diff
@@ -7,7 +7,7 @@ def seed!(verbose=false)
     ReportType.find_or_create_by!(abbr:'R003', full_name: 'Business unit report', class_name: 'Stats::R003BusinessUnitPerformanceReport' , custom_report: true, seq_id: 200)
     ReportType.find_or_create_by!(abbr:'R004', full_name: 'Cabinet Office report', class_name: 'Stats::R004CabinetOfficeReport', custom_report: false, seq_id: 400)
     ReportType.find_or_create_by!(abbr:'R005', full_name: 'Monthly report', class_name: 'Stats::R005MonthlyPerformanceReport',custom_report: true, seq_id: 300)
-    ReportType.find_or_create_by!(abbr:'R006', full_name: 'Business unit map', class_name: 'Stats::R006KiloMap', custom_report: false, seq_id: 9999)
+    ReportType.find_or_create_by!(abbr:'R006', full_name: 'Business Unit Map', class_name: 'Stats::R006KiloMap', custom_report: false, seq_id: 9999)
```

Comment:
Don't bother changing this back, but for future reference the GDS standard is not to use capital letters for each word, so Business unit not Business Unit

Reply:
I changed this because this was failing as something else creates the report with `Business Unit Map`, I think maybe from a migration.